src/Cake.Core/ICakeRuntime.cs

```diff
@@ -13,10 +14,15 @@ namespace Cake.Core
     public interface ICakeRuntime
     {
         /// <summary>
-        /// Gets the target .NET framework version that the current AppDomain is targeting.
+        /// Gets the build-time .NET framework version that is being used.
+        /// </summary>
+        FrameworkName BuiltFramework { get; }
+
+        /// <summary>
+        /// Gets the current executing .NET Runtime.
         /// </summary>
         /// <returns>The target framework.</returns>
-        FrameworkName TargetFramework { get; }
+        Runtime Runtime { get; }
```

Comment:
Breaking changes... We sould also bump the `breaking change` version [here](https://github.com/cake-build/cake/blob/develop/src/Cake.Core/Constants.cs#L11)

Reply:
Fixed